result/bmpserver.yaml

```diff
@@ -0,0 +1,129 @@
+          x-field-uid: 2
+        system_name:
+          description: >-
+            The Information sysName MIB-II [RFC1213] object information carried in Information field of 
+            Information TLV in Initiation message.
+          type: string
+          x-field-uid: 3
+        system_description:
+          description: >-
+            The Information sysDesc MIB-II [RFC1213] object information carried in Information field of 
+            Information TLV in Initiation message.
+          type: string
+          x-field-uid: 4
+        termination_string:
+          description: >-
+            The UTF-8 string information carried in Information field of Information TLV in Termination message. It MAY be used 
+            to provide further detail for any of the defined reasons. Multiple String TLVs may be included in the message.
+          type: array
+          items:
+            type: string
+          x-field-uid: 5        
+        termination_reason:
+          description: >-
+            The Information field contains a 2-byte code indicating the reason that the connection was terminated.
+          type: array
+          items:
+            type: string
+            x-enum:
+              administratively_closed:
+                x-field-uid: 1
+              not_specified:
+                x-field-uid: 2
+              out_of_resource:
+                x-field-uid: 3
+              redundant_connection:
+                x-field-uid: 4
+              administratively_closed_permanently:
+                x-field-uid: 5
+          x-field-uid: 6
+        peers:
+          type: array
+          items:
+            $ref: '#/components/schemas/BmpServerPeer.State'
+          x-field-uid: 7
+    BmpServerPeer.State:
+      description: >-
+        The Peer Up message is used to indicate that a peering session has come up 
+        (i.e., has transitioned into the Established state). 
+      type: object
+      properties:
+        local_address:
+          description: >-
+            The IPv4 or IPv6 local address associated with the Bgp peer.
+          type: string
+          x-field-uid: 1
+        remote_address:
+          description: >-
+            The IPv4 or IPv6 remote address associated with the Bgp peer.
+          type: string
+          x-field-uid: 2
+        local_port:
+          description: >-
+            The local port number associated with the peering TCP session, or 0 if no TCP session actually exists.
+          type: integer
+          format: uint32
+          x-field-uid: 3
+        remote_port:
+          description: >-
+            The remote port number associated with the peering TCP session, or 0 if no TCP session actually exists.
+          type: integer
+          format: uint32
+          x-field-uid: 4
+        peer_bgp_id:
+          description: >-
+            The BGP Identifier of the peer.
+          type: string
+          x-field-uid: 5
+        peer_down_reason:
```

Comment:
Again , should this be within some current state and if down then show reason and if up show per peer info ?